andela/bp-esa-frontend
# added notification for connection error and some refactoring

#### What does this PR do?
- add notify to ReportPage index.jsx
- adds new methods to refactor some statements
#### Description of Task to be completed?
shows notification on server error

#### How should this be manually tested?
1. On browser, goto the localhost:3000, set the REACT_APP_BACKEND_URL to an unknown server url
2. when the browser tries to display the table data and fails due to failure in connecting to the server configured, you should see a notification saying unable to connect to the server


#### What are the relevant pivotal tracker stories?
#165172096

#### Screenshots (If applicable)

<img width="1440" alt="Screenshot 2019-04-08 at 5 09 00 AM" src="https://user-images.githubusercontent.com/8375520/55698313-1a831500-59bd-11e9-9c5f-b41ef2df5205.png">
